apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -1,56 +1,125 @@
+import { Link } from "react-router-dom";
+
+import {
+	blueStars,
+	carImage,
+	orangeImage,
+	yellowStars,
+} from "~/assets/img/sign-up/sign-up.img.js";
 import { Button, Input } from "~/libs/components/components.js";
+import { AppRoute } from "~/libs/enums/app-route.enum.js";
 import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
 import {
+	type SignUpFormValidationSchema,
 	type UserSignUpRequestDto,
-	userSignUpValidationSchema,
+	userSignUpValidationSchemaExtended,
 } from "~/modules/users/users.js";
 
 import { DEFAULT_SIGN_UP_PAYLOAD } from "./libs/constants.js";
+import css from "./sign-up-form.module.css";
 
 type Properties = {
 	onSubmit: (payload: UserSignUpRequestDto) => void;
 };
 
 const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
-	const { control, errors, handleSubmit } = useAppForm<UserSignUpRequestDto>({
-		defaultValues: DEFAULT_SIGN_UP_PAYLOAD,
-		validationSchema: userSignUpValidationSchema,
-	});
-
+	const { control, errors, handleSubmit } =
+		useAppForm<SignUpFormValidationSchema>({
+			defaultValues: { ...DEFAULT_SIGN_UP_PAYLOAD, confirmPassword: "" },
+			validationSchema: userSignUpValidationSchemaExtended,
+		});
 	const handleFormSubmit = useCallback(
 		(event_: React.BaseSyntheticEvent): void => {
-			void handleSubmit(onSubmit)(event_);
+			void handleSubmit((data) => {
+				// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
```suggestion
				const { confirmPassword: _, ...payloadToSend } = data;
```
it tells linter that var can be unused